Library/Formula/sshguard.rb

```diff
@@ -19,20 +21,12 @@ class Sshguard < Formula
     sha1 "68cd0910d310e4d23e7752dee1b077ccfe715c0b"
   end
```

Comment:
Pulling this as-is for now but if this isn't getting merged then I think we should think about killing it.